### ISSUE TITLE
[Request] Max seed (time / ratio)

### ISSUE THREAD

**Issue by Phr33d0m** [jfurrow/flood#371]
Hi,

To exemplify better have a look at this screenshot from KTorrent (_note: the default seed time is in hours_):
![image](https://cloud.githubusercontent.com/assets/1591486/26532626/2a1560e6-4406-11e7-90f2-979d9bd53356.png)

Something like this would be an awesome asset to flood. An addition to the screenshot would be what to do with the torrent once the max seeded time was reached or the max ratio was reached:
 - Stop the torrent
 - Delete the torrent
 - Delete the torrent with data

Some example use cases after reaching max seed:
 - Stop the torrent on our home computers
 - Delete the torrent on seedboxes
 - Delete the torrent with data on replication seedboxes (this is my personal case)

Regards.

**Comment by jfurrow** [comment]
Yeah, I agree Flood needs this, in addition to global throttle scheduling.
